server/src/main/java/org/candlepin/dto/api/v1/EnvironmentContentDTO.java

```diff
@@ -0,0 +1,188 @@
+/**
+ * Copyright (c) 2009 - 2017 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.dto.api.v1;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+import io.swagger.annotations.ApiModel;
+
+/**
+ * A DTO representation of the Environment Content entity
+ */
+@ApiModel(parent = TimestampedCandlepinDTO.class, description = "DTO representing an environment content")
+public class EnvironmentContentDTO extends TimestampedCandlepinDTO<EnvironmentContentDTO> {
```

Comment:
This class could probably be a private class within EnvironmentDTO. We could also drop its ID and timestamps from the output, as none of them are important to end users. See ProductDTO.ProductContentDTO for an similar example.

Reply:
done